### PR TITLE
created login/sign-up page

### PR DESCRIPTION
**Pull request by agrawalshanu708**
created a sign-up & login-page
ecom-login
ecom-sign

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,18 +3,25 @@ import logo from "./logo.png";
 import { Routes, Route, Link } from "react-router-dom";
 import Mockman from "mockman-js"
 import {Navbar,Footer} from "./components/index"
-import {Home} from "./pages/Home/homepage"
-import {ProductListing} from "./pages/ProductListing/productListing"
+import {Home,ProductListing,Login,Signup} from "./pages/index"
```
Comment (harshitpaliwal95) [inline src/App.js:6]: its good u are importing pages from single file

File: src/pages/Auth/login/login.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      <p>Enter your Credentials to access your account</p>
+      
+      <div className="login-button-box">
+      <button className="login-with-google-button">Login with Google</button>
+      </div>
+
+      <hr className="login-hr"/>
+
+
+      <form action="">
```
Comment (Sharmaryan) [inline src/pages/Auth/login/login.jsx:24]: action attribute is empty

File: src/pages/Auth/login/login.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      <label htmlFor= "user-pass" className="label-text" type="password">Enter password</label>
+      <label className="forget-pass">Forgot Password?</label>
+      </div>
+      <input id="user-pass"  className="login-input" type="text" placeholder="Enter your password" />
+      </div>
+
+      <div className="remember-me-box">
+      <input  type="checkbox" id="remember-me" />
+      <label className="label-remember" forhtml ="remember-me">Remember me</label>
+      </div>
```
Comment (Sharmaryan) [inline src/pages/Auth/login/login.jsx:42]: you have written forhtml , instead of htmlFor

File: src/App.css [hunk 4]
```diff
@@ -1,53 +1,14 @@
...
-.brand-description {
-  font-size: 1.4rem;
+.footer{
+  align-self: stretch;
 }
-
-.links {
-  display: flex;
+html{
+  font-size:60%
```
Comment (aurobindoGupta) [inline src/App.css:12]: rem?

File: src/pages/index.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+export {Home} from "./Home/homepage";
+export {ProductListing} from "./ProductListing/productListing"
+export {Login} from "./Auth/login/login"
+export {Signup} from "./Auth/signup/signup"
```
Comment (harshitpaliwal95) [inline src/pages/index.js:4]: can create single file for importing same comp here too

**Review by harshitpaliwal95 — COMMENTED** [review]
overall looks good u can merge

File: public/index.html [hunk 6]
```diff
@@ -17,6 +17,7 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <link rel="stylesheet" href="https://crabui.vercel.app/mainstyle.css">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css%22">
     <!--
       Notice the use of %PUBLIC_URL% in the tags above.
       It will be replaced with the URL of the `public` folder during the build.
```
Comment (Sharmaryan) [inline public/index.html:23]: you can remove these comments

**Review by Sharmaryan — APPROVED** [review]
suggest some small changes

File: src/pages/Auth/login/login.css [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+}
+.login-hr{
+    margin: 2rem 0 0 0;
+}
+.login-pic img{
+    width: 100%;
+    height: 100%
+}
+.blue-dot{
+    color: #22D3EE;
```
Comment (aurobindoGupta) [inline src/pages/Auth/login/login.css:14]: variable

File: src/pages/Auth/login/login.css [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+
+.login-with-google-button,.login-submit-btn{
+    min-width: 40%;
+    height:4rem;
+    background-color:transparent;
+    margin: 1rem 0;
+    padding:1rem;
+}
+.login-with-google-button{
+    border: 1px solid #22D3EE;
```
Comment (aurobindoGupta) [inline src/pages/Auth/login/login.css:29]: variable

**Review by aurobindoGupta — APPROVED** [review]
nice work

**Pull request merged**